hedera-node/src/main/java/com/hedera/services/ledger/MerkleAccountScopedCheck.java

```diff
@@ -74,6 +80,12 @@ public MerkleAccountScopedCheck setBalanceChange(final BalanceChange balanceChan
 		return this;
 	}
 
+	public MerkleAccountScopedCheck setNftsLedger(
```

Comment:
We could just inject this in the constructor, since a given `TransferLogic` only ever has one `nftsLedger` instance.